x/incentives/keeper/distribute.go

```diff
@@ -462,6 +462,31 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 	return totalDistributedCoins, nil
 }
 
+// TODO: consider performance. Is this too inefficient?
+// func (k Keeper) AllocateAcrossGauges(ctx sdk.Context, groupGaugeIds []uint64) (sdk.Coins, error) {
+// 	// Loop through group gauges, and for each:
+// 		// Figure out splitting policy
+// 		// if VolumeSplitting (enum 0), run AllocateByVolume
```

Comment:
TODO: Check if the group gauge coins is > the addToGauge rewards amount

Reply:
Also forgot to ask: Is the idea that the lower the volume of the pool, the higher the incentive split reward it gets? so that rich doesnot get richer

Reply:
> TODO: Check if the group gauge coins is > the addToGauge rewards amount

Hmm not sure I follow – the amount to add to gauge should always be a slice of the gauge coins being distributed

Reply:
> Also forgot to ask: Is the idea that the lower the volume of the pool, the higher the incentive split reward it gets? so that rich doesnot get richer

Exact opposite – we want the incentives to be going to the pools that have the most usage to get the positive flywheel going (more vol -> more liq -> more vol). I think of it like volume signals demand, so incentives help supply get there faster